compute/apiv1/smoke_test.go

```
@@ -0,0 +1,290 @@
+		&disk,
+	}
+	instance := &computepb.Instance{
+		Name: &name,
+		Description: &description,
+		MachineType: &machineType,
+		Disks: disks,
+		NetworkInterfaces: interfaces,
+	}
+
+	createRequest := &computepb.InsertInstanceRequest{
+		Project: projectId,
+		Zone: defaultZone,
+		InstanceResource: instance,
+	}
+
+	insert, err := c.Insert(ctx, createRequest)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+
+	waitZonalRequest := &computepb.WaitZoneOperationRequest{
+		Project: projectId,
+		Zone: defaultZone,
+		Operation: insert.GetName(),
+	}
+	_, err = zonesClient.Wait(ctx, waitZonalRequest)
+	if err != nil {
+		return
+	}
+	fmt.Printf("Inserted instance named %s\n", name)
+	defer ForceDeleteInstance(name, ctx, c)
+
+	getRequest := &computepb.GetInstanceRequest{
+		Project: projectId,
+		Zone: defaultZone,
+		Instance: name,
+	}
+	get, err := c.Get(ctx, getRequest)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+	assert.Equal(t, name, get.GetName())
+	assert.Equal(t, "тест", get.GetDescription())
+
+	listRequest := &computepb.ListInstancesRequest{
+		Project: projectId,
+		Zone: defaultZone,
+	}
+
+	list, err := c.List(ctx, listRequest)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+	items := list.GetItems()
+	found := false
+	for _, element := range items {
+		if element.GetName() == name {
+			found = true
+		}
+	}
+	if found == false{
+		t.Error("Couldn't find the instance in list response")
+	}
+
+	deleteInstanceRequest := &computepb.DeleteInstanceRequest{
+		Project: projectId,
+		Zone: defaultZone,
+		Instance: name,
+	}
+	_, err = c.Delete(ctx, deleteInstanceRequest)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+}
+
+func ForceDeleteInstance(name string, ctx context.Context, client *InstancesClient){
+	deleteInstanceRequest := &computepb.DeleteInstanceRequest{
+		Project: projectId,
+		Zone: defaultZone,
+		Instance: name,
+	}
+	_, err := client.Delete(ctx, deleteInstanceRequest)
+	if err != nil {}
```

Should we be swallowing the error? If so, just remove the `_, err :=` and this line. If you think it should be reported, please pass in the `*testing.T` instance.

---

We should swallow the error here as it works like tear down.